net/gftp/Portfile

```diff
@@ -1,48 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem      1.0
+PortGroup       github 1.0
+PortGroup       openssl 1.0
 
-name            gftp
-version         2.0.18
-revision        9
+github.setup    masneyb gftp 2.9.1b
+revision        0
 categories      net
 license         GPL-2+
-platforms       darwin
 maintainers     nomaintainer
 description     GTK+ FTP client
 
 long_description \
-    gFTP is a free multithreaded file transfer client for *NIX based machines.
+                gFTP is a free multithreaded file transfer client for *NIX based machines.
 
 homepage        http://www.gftp.org/
-master_sites    ${homepage} \
-                ftp://www.gftp.org/pub/gftp/
 
-use_bzip2       yes
+checksums       rmd160  523938683564a045305338a3ef03ae8a1a56ee58 \
+                sha256  c0198f26b05b1156b37f1bc51f1359490e031a43e3acf6d145235063be3cf93c \
+                size    1557295
 
-checksums       md5 44b19442d06f6dc68c7211d66029d826 \
-                sha1 c2468f1ff1fe978a5fab3a198e513558f3b48f5a \
-                rmd160 0e0daea8543ae822b25635647360193362a2648d
+patchfiles      patch-openpty.diff
 
-depends_build   port:pkgconfig
-depends_lib     path:lib/pkgconfig/gtk+-2.0.pc:gtk2 \
+depends_build-append \
+                port:autoconf \
+                port:automake \
+                port:libtool \
+                port:pkgconfig
+depends_lib-append \
+                path:lib/pkgconfig/gtk+-2.0.pc:gtk2 \
                 port:gettext \
```

Comment:
If this project actually links with `gettext` - and it would be good to confirm, via `otool` - then the lib dep should be `gettext-runtime`. Along with a build dep on `gettext`.

Reply:
@mascguy 
```
otool -L /opt/local/bin/gftp-gtk
/opt/local/bin/gftp-gtk:
	/opt/local/lib/libgtk-x11-2.0.0.dylib (compatibility version 2401.0.0, current version 2401.33.0)
	/opt/local/lib/libgdk-x11-2.0.0.dylib (compatibility version 2401.0.0, current version 2401.33.0)
	/opt/local/lib/libpangocairo-1.0.0.dylib (compatibility version 5001.0.0, current version 5001.7.0)
	/opt/local/lib/libXrender.1.dylib (compatibility version 5.0.0, current version 5.0.0)
	/opt/local/lib/libXinerama.1.dylib (compatibility version 2.0.0, current version 2.0.0)
	/opt/local/lib/libXi.6.dylib (compatibility version 8.0.0, current version 8.0.0)
	/opt/local/lib/libXrandr.2.dylib (compatibility version 5.0.0, current version 5.0.0)
	/opt/local/lib/libXcursor.1.dylib (compatibility version 2.0.0, current version 2.2.0)
	/opt/local/lib/libXcomposite.1.dylib (compatibility version 2.0.0, current version 2.0.0)
	/opt/local/lib/libXdamage.1.dylib (compatibility version 3.0.0, current version 3.0.0)
	/opt/local/lib/libXfixes.3.dylib (compatibility version 5.0.0, current version 5.0.0)
	/opt/local/lib/libX11.6.dylib (compatibility version 11.0.0, current version 11.0.0)
	/opt/local/lib/libXext.6.dylib (compatibility version 11.0.0, current version 11.0.0)
	/opt/local/lib/libatk-1.0.0.dylib (compatibility version 23810.0.0, current version 23810.1.0)
	/opt/local/lib/libcairo.2.dylib (compatibility version 11707.0.0, current version 11707.0.0)
	/opt/local/lib/libgdk_pixbuf-2.0.0.dylib (compatibility version 4201.0.0, current version 4201.9.0)
	/opt/local/lib/libgio-2.0.0.dylib (compatibility version 7401.0.0, current version 7401.0.0)
	/opt/local/lib/libpangoft2-1.0.0.dylib (compatibility version 5001.0.0, current version 5001.7.0)
	/opt/local/lib/libpango-1.0.0.dylib (compatibility version 5001.0.0, current version 5001.7.0)
	/opt/local/lib/libgobject-2.0.0.dylib (compatibility version 7401.0.0, current version 7401.0.0)
	/opt/local/lib/libglib-2.0.0.dylib (compatibility version 7401.0.0, current version 7401.0.0)
	/opt/local/lib/libintl.8.dylib (compatibility version 11.0.0, current version 11.0.0)
	/opt/local/lib/libharfbuzz.0.dylib (compatibility version 50101.0.0, current version 50101.0.0)
	/opt/local/lib/libfontconfig.1.dylib (compatibility version 14.0.0, current version 14.0.0)
	/opt/local/lib/libfreetype.6.dylib (compatibility version 25.0.0, current version 25.3.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 125.2.11)
	/usr/lib/libutil.dylib (compatibility version 1.0.0, current version 1.0.0)
	/opt/local/lib/libgthread-2.0.0.dylib (compatibility version 7401.0.0, current version 7401.0.0)
	/opt/local/libexec/openssl3/lib/libssl.3.dylib (compatibility version 3.0.0, current version 3.0.0)
	/opt/local/libexec/openssl3/lib/libcrypto.3.dylib (compatibility version 3.0.0, current version 3.0.0)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 550.44.0)
```

```
otool -L /opt/local/bin/gftp-text 
/opt/local/bin/gftp-text:
	/opt/local/lib/libglib-2.0.0.dylib (compatibility version 7401.0.0, current version 7401.0.0)
	/opt/local/lib/libintl.8.dylib (compatibility version 11.0.0, current version 11.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 125.2.11)
	/usr/lib/libutil.dylib (compatibility version 1.0.0, current version 1.0.0)
	/opt/local/lib/libreadline.8.dylib (compatibility version 8.1.0, current version 8.1.0)
	/opt/local/lib/libncurses.6.dylib (compatibility version 6.0.0, current version 6.0.0)
	/opt/local/libexec/openssl3/lib/libssl.3.dylib (compatibility version 3.0.0, current version 3.0.0)
	/opt/local/libexec/openssl3/lib/libcrypto.3.dylib (compatibility version 3.0.0, current version 3.0.0)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 550.44.0)
```

Reply:
Thanks, so there's a definite lib dep. So make the changes mentioned.